docs/appendices/release-notes/unreleased.rst

```diff
@@ -225,6 +225,9 @@ Performance improvements
 Fixes
 =====
 
+- Limit the output of COPY FROM RETURN SUMMARY in the presence of errors to
```

Comment:
This is kind a breaking change and not a fix or do we plan to backport it?

Reply:
I don't know, I suppose it fixes the crash/Admin UI issue, should it be backported?